controllers/platform/k8s.go

```diff
@@ -243,7 +250,7 @@ func createConfigMap(ctx context.Context, client client.Client, platform *operat
 			Name:      ps.GetServiceCmName(),
 			Namespace: platform.Namespace,
 			Labels: map[string]string{
-				workflowproj.LabelApp: platform.Name,
+				workflowproj.LabelService: platform.Name,
```

Comment:
shouldn't this have both labels set? also, this line should remain as-is... and the new label added. this should match the other objects.

Reply:
yes, both labels should be added here.

Reply:
```
→ kubectl get cm  -n sonataflow-infra -l app=sonataflow-platform   --show-labels
NAME                                           DATA   AGE    LABELS
sonataflow-platform-data-index-service-props   1      4m9s   app=sonataflow-platform,sonataflow.org/service=sonataflow-platform-data-index-service
sonataflow-platform-jobs-service-props         1      4m8s   app=sonataflow-platform,sonataflow.org/service=sonataflow-platform-jobs-service
```